src/main/resources/dbreplica/migration/V1.0.13__create_indexes_to_replica_db.sql

```diff
@@ -0,0 +1,31 @@
+-- tasks indexes
+
+CREATE INDEX idx_t_task_id ON cft_task_db.tasks (state);
+
+-- task_history indexes
+
+CREATE INDEX idx_th_created_update_id ON cft_task_db.task_history (created, update_id);
+CREATE INDEX idx_th_updated_update_id ON cft_task_db.task_history (updated, update_id);
+
+-- reportable_task indexes
+
+CREATE INDEX idx_rt_created_date_update_id ON cft_task_db.reportable_task (created_date, update_id);
+CREATE INDEX idx_rt_due_date_update_id ON cft_task_db.reportable_task (due_date, update_id);
+CREATE INDEX idx_rt_state_rep ON cft_task_db.reportable_task (state);
+CREATE INDEX idx_rt_final_state_label ON cft_task_db.reportable_task (final_state_label);
+CREATE INDEX idx_is_within_sla ON cft_task_db.reportable_task (is_within_sla);
```

Comment:
Not too sure, if you wanted idx_rt as prefix for rest of the indexes on reportable_task table as well ?

Reply:
Yup good point!  Have pushed a change.